### ISSUE TITLE
Norway limits amount of tests to groups where it will have an impact. Cases number will be wrong

### ISSUE THREAD

**Issue by SomePlayer** [CSSEGISandData/COVID-19#872]
As Norway now only tests groups where they estimate it will have an impact. And the test has a purpose. 

As Line Vold at Norwegian Institute of Public Health says that people in quarantine are no longer tested. Because the response would be to go into quarantine, and they are already in (home) quarantine. 
Source: https://www.vg.no/nyheter/innenriks/i/LAxGRV/jobber-med-aa-utvide-testkapasiteten-vi-tester-der-det-trengs-mest

I suggest that the data reflect that there maybe dark numbers for Norway. And recover/death ratio will be obscured, when most of the tested cases are serious cases.

Maybe a statistic for hospitalized, or amount of people in quarantine can reflect how widespread the virus is in an area.

**Comment by JiPiBi** [comment]
I agree, quantities of people in quarantine and in ICU should be very interesting , but that data (apart in Italy everyday)  are not broadcasted : strategy or inefficiency ?

The number of tests made  by day has no real signification,  for example in Italy because the same person can be tested several times during some days and in France it is not made until you have severe signs of illness. 

In France , the count of recovered is not updated from the beginning : only 12 .....
